src/ome_zarr_models/v04/image.py

```diff
@@ -67,19 +85,53 @@ class TranslationTransform:
     type: Literal["translation"]
     translation: Sequence[float]
 
+    @classmethod
+    def _from_json(cls, json: dict) -> Self:
+        return cls(type="translation", translation=json["translation"])
+
 
 @dataclass(frozen=True, slots=True, kw_only=True)
-class Dataset:
+class Dataset(ToFromJSON):
+    """
+    A single dataset.
+
+    Parameters
+    ----------
+    path :
+        Path to dataset.
+    coordinateTransformations :
+        Coordinate transformations for dataset.
+
+    References
+    ----------
+    https://ngff.openmicroscopy.org/0.4/index.html#multiscale-md
+    """
+
     path: str
     coordinateTransformations: (
         tuple[ScaleTransform] | tuple[ScaleTransform, TranslationTransform | str]
     )
 
+    @classmethod
+    def _from_json(cls, json) -> Self:
```

Comment:
This variable name might cause name collisions if we do `import json`

Reply:
ooh yes, I will change